scripts/dockerbuild

```diff
@@ -28,6 +31,10 @@ fi
 if [[ "$1" = hugo ]]; then
     _DEFAULT_IMAGE=ghcr.io/scilifelabdatacentre/swg-hugo-site
     _DEFAULT_DOCKERFILE=docker/hugo.dockerfile
+    declare -a _BUILD_ARGS
```

Comment:
The `_BUILD_ARGS` declaration could be moved at the top scope. But more importantly, perhaps: should we provide a default value ? For example:
```bash
"HUGO_ENV_ARG=${SWG_HUGO_ENV_ARG:-$(git rev-parse HEAD)}"
```
I also like `git describe` but it shoud be used with some care here since it fails without a least one tag earlier in the history.

Reply:
I've moved the `_BUILD_ARGS` to the top. 

The default currently says "local dev build" to reflect that you might have some uncommitted changes you are testing.